envtrace

```diff
@@ -2,7 +2,7 @@
 
 ENVTRACE_ROOT="$(readlink -f $(dirname "$0"))"
 
-export LD_PRELOAD=${ENVTRACE_ROOT}/envtrace-helper.so
+export LD_PRELOAD=${ENVTRACE_ROOT}/envtrace-helper.so:/lib64/libdl.so.2:${LD_PRELOAD}
```

Comment:
`/lib64/libdl.so.2` seems like a really specific path and version that may break in future upgrades.
Does plain `libdl.so` work?

Reply:
/lib/libdl.so does not exist on my machine (/lib/libdl.so.2 does). I changed the LD_PRELOAD path to /lib64/libdl.so with seemingly no negative change in functionality.